PlanetNode/GraphTypes/PlanetNodeMutation.cs

```diff
@@ -17,15 +17,17 @@ public PlanetNodeMutation(BlockChain<PolymorphicAction<PlanetAction>> blockChain
     {
         Field<TransactionMutation<PolymorphicAction<PlanetAction>>>(
             "transaction",
+            description: "Adds transaction to the pending list so that a next Block to be mined may contain given transaction.",
             resolve: context => new { }
         );
 
         Field<TransactionType<PolymorphicAction<PlanetAction>>>(
             "transferAsset",
+            description: "Mutates a new transaction about transfer and stage the transaction.",
             arguments: new QueryArguments(
-                new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "recipient" },
-                new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "amount" },
-                new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "privateKeyHex" }
+                new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "recipient", Description = "Argument adress is the recipient's public key." },
+                new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "amount", Description = "Argument amount is transfer amount. The unit of transfer amount is PNG." },
+                new QueryArgument<NonNullGraphType<StringGraphType>> { Name = "privateKeyHex", Description = "Argument privateKeyHex is the sender's hexadecimal private key used for signing." }
```

Comment:
- Redundant the first words “Argument”
- Long lines into multiple lines

```suggestion
                new QueryArgument<NonNullGraphType<StringGraphType>>
                {
                	    Name = "recipient",
                	    Description = "The recipient's 40-hex address.",
                	},
                new QueryArgument<NonNullGraphType<StringGraphType>>
                {
                	    Name = "amount",
                	    Description = "The amount to transfer in PNG.",
                	},
                new QueryArgument<NonNullGraphType<StringGraphType>>
                {
                	    Name = "privateKeyHex",
                	    Description = "A hex-encoded private key of the sender.  A made " +
                	        "transaction will be signed using this key.",
                	}
```